packages/core/lib/utils.d.ts

```diff
@@ -0,0 +1,19 @@
+import Segment = require('./segments/segment');
+
+export function getCauseTypeFromHttpStatus(status: number): 'error' | 'fault' | undefined;
```

Comment:
`status` should be a `string`, not a `number`

Reply:
I was basing `number` off https://nodejs.org/api/http.html#http_response_statuscode and  https://github.com/aws/aws-xray-sdk-node/blob/180e70bbab811d0beb994b2804e4b832cbab8698/packages/core/test/unit/utils.test.js#L12-L20

But does `getCauseTypeFromHttpStatus` definitely get called with a `string` too? It could be `number | string`?

Reply:
the function signature expects a string, but seeing how the status code is a number, I'd agree that a `number | string` is the safest bet.